src/common/tcp_info.cc

```diff
@@ -0,0 +1,121 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
```

Comment:
:(

Reply:
What would be the right emacs/vim header?
GH matches about 3k files with this.

I tend to use clang-format instead (with https://github.com/s3gw-tech/s3gw-ceph/blob/s3gw/.clang-format)

Reply:
vi is never the right answer ;)

Reply:
so I guess this review comment can be resolved, given there's hardly an tangible action to do other than potentially kill vi?